tsconfig.base.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "@tsconfig/recommended/tsconfig.json",
+  "compilerOptions": {
+    "allowJs": true,
+    "checkJs": true,
+    "lib": ["ESNext"],
+    "module": "NodeNext",
+    "noEmit": true,
```

Comment:
We don't want TypeScript to compile anything yet, including source maps `*.js.map` and [types `*.d.ts`](https://github.com/alphagov/govuk-frontend/issues/2835)